apps/payments/payments-next/app/global.css

```diff
@@ -0,0 +1,400 @@
+html {
+  -webkit-text-size-adjust: 100%;
+  font-family: ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont,
+    Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif,
+    Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji;
```

Comment:
nit: do we need all these fonts?
Ah, this looks like a boilerplate CSS reset&hellip;